### DISCUSSION TITLE
1. THANKS 2. Translations

### DISCUSSION THREAD

**Discussion by JVariance** [SuhasDissa/Food-E-App#41] [General]
Dude, this app is wonderful!

Maybe you could translate the E-Number Names and Descriptions into other languages via DeepL or another AI?

**Comment by SuhasDissa** [comment]
Thanks for your suggestion! I appreciate your feedback.

Unfortunately, there's no easy way to store multiple locales in Android databases right now.
 I'll look into the possibility of using AI to do the translations, but in the meantime, you can **copy the additive details and paste them into your favorite translation app**.

I hope this helps! Let me know if you have any other questions.

**Comment by JVariance** [comment]
It's there, thanks 😍

**Reply by SuhasDissa** [reply]
You're welcome 🤗
